modules/cf-core/src/node.ts

```diff
@@ -43,7 +39,6 @@ export class Node {
   private readonly outgoing: EventEmitter;
 
   private readonly protocolRunner: ProtocolRunner;
-  private readonly networkContext: NetworkContext;
```

Comment:
is there a reason not to save the network context to the node?

Reply:
thinking about if there is a case where a default counterfactual context is used when we try to PR this upstream

Reply:
Default context was kinda useless imo (hence why I killed it).
It wasn't being used anywhere and contract addresses are too important to just quietly fall back on defaults that are probably out of date. I'd rather be clear: either the layer above gives us addresses that they want to use, or this layer should freak out.